### PR TITLE
Prepare 1.0

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] Version dependencies
* [x] Update changelog
* [ ] Backport cliquet docs changes (**ETags ?**)
* [x] Update version in docs/conf.py
* [ ] Tag vX.Y.Z
* [ ] Close milestone in Github
* [ ] Publish on Pypi
* [ ] Add entry in Github release page
* [ ] Create next milestone in Github
* [ ] Send mail to ML
* [ ] Blog about it

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,10 +4,42 @@ Changelog
...
+**New features**
+
+- Added notion of bucket, user groups and collections (#48, #58)
+- Bucket, collections and records can now have permissions (#59)
+
+**Breaking changes**
+
+- Updated *Cliquet* to 2.0, which introduces a lot of breaking changes
+  (`see changelog <https://github.com/mozilla-services/cliquet/releases/2.0.0>`_)
+- Firefox Accounts is now a dependency anymore and should be installed and
```
Comment (Natim) [inline CHANGELOG.rst]: is not

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,10 +4,42 @@ Changelog
...
 
 
-1.0.0 (unreleased)
+1.0.0 (2015-06-16)
 ==================
 
-- Nothing changed yet.
+**New features**
+
+- Added notion of bucket, user groups and collections (#48, #58)
```
Comment (ametaireau) [inline CHANGELOG.rst]: buckets

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,10 +4,42 @@ Changelog
...
+- Bucket, collections and records can now have permissions (#59)
```
Comment (ametaireau) [inline CHANGELOG.rst]: Buckets

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,10 +4,42 @@ Changelog
...
+  (`see changelog <https://github.com/mozilla-services/cliquet/releases/2.0.0>`_)
+- Firefox Accounts is now a dependency anymore and should be installed and
+  included explictly using the python package ``cliquet-fxa``
+  (`see documentation <https://github.com/mozilla-services/cliquet-fxa/>`_)
+- API is now served under ``/v1``
+- Collections are now managed by bucket, and not by user anymore (#44)
+
+.. note::
+
+    A list records cannot be manipulated until its parents objects (bucket and
```
Comment (ametaireau) [inline CHANGELOG.rst]: A list of records

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,10 +4,42 @@ Changelog
...
+    collection) are created.
+
+Settings
+
+- ``cliquet.permission_backend`` and ``cliquet.permission_url`` should be configured
```
Comment (ametaireau) [inline CHANGELOG.rst]: Are now

File: CHANGELOG.rst [hunk 6]
```diff
@@ -4,10 +4,42 @@ Changelog
...
+  to use PostgreSQL instead of *Redis* (see default :file:`config/kinto.ini`)
+- ``cliquet.basic_auth_enabled`` is now deprecated (see *Cliquet* docs to enable
```
Comment (ametaireau) [inline CHANGELOG.rst]: Link to http://cliquet.readthedocs.org/en/latest/reference/configuration.html#basic-auth

**Comment by ametaireau** [comment]
LGTM with nitpicks.

**Pull request merged**